### PR TITLE
Frontend graph

### PR DESCRIPTION
**Pull request by s-block**

### CODE REVIEW

File: dashboard/apps/prototype/static/prototype/js/main.js [hunk 1]
```diff
@@ -191,24 +191,24 @@ var testRequest = {
   end_date: '2016-05-23'
 };
 
+function mapFloats(list, key) {
```
Comment (cliffxuan) [inline dashboard/apps/prototype/static/prototype/js/main.js]: i'm working on the refactoring of this, and these are reorganised in my new code. might be better to leave this change out for the time being.

File: dashboard/apps/prototype/static/prototype/js/main.js [hunk 2]
```diff
@@ -253,7 +253,7 @@ function plotProject(project) {
     mode: 'lines',
     yaxis: 'y2',
     marker: {
-      color: '#F0E891'
+      color: '#FFBF47'
```
Comment (cliffxuan) [inline dashboard/apps/prototype/static/prototype/js/main.js:256]: the color change is good to be merged.

File: dashboard/apps/prototype/static/prototype/js/main.js (hunk 1 above)
Comment (cliffxuan) [inline dashboard/apps/prototype/static/prototype/js/main.js]: the problem this function is trying to address is captured in this function here in my wip branch:
https://github.com/ministryofjustice/moj-product-dashboard/blob/feature/unit-tests/dashboard/apps/prototype/static/prototype/js/main.js#L37

Comment (s-block) [inline dashboard/apps/prototype/static/prototype/js/main.js]: cool

**Pull request merged**